### DISCUSSION TITLE
Where should I write my config to avoid them been covered by other plugins?

### DISCUSSION THREAD

**Discussion by Vescrity** [NvChad/NvChad#2892] [Q&A]
As the title. I noticed that my config (such as tabstop) often be covered by such as `ftplugin` .  Where should I config them to avoid them be covered?  Even I set them when I edit the file, the setting still doesn't change.

**Comment by siduck** [comment]
@Vescrity ~/.config/nvim/ftplugin? or autocmd on filetype event
